src/ima.js

```diff
@@ -1181,7 +1183,13 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     this._adsManager.addEventListener(this._sdk.AdEvent.Type.CONTENT_PAUSE_REQUESTED, adEvent => {
       if (this._playAdByConfig() || this._firstOfAdPod) {
         this._firstOfAdPod = false;
+        this._adBreakStarted = true;
```

Comment:
1. need to reset it on `CONTENT_RESUME_REQUESTED` to false
2. actually you can use `this.player.ads.isAdBreak()` instead of this flag